[Editorial] Add the definitions from the Demon's Dictionary

mmastrac opened the pull request:
This commit adds the words originally in the Demon's Dictionary column from Dec 11 1875, merging them where duplicates exist. For many of the words, an ordered list was created for the word and the two definitions were moved to list items. In other cases, a light editorial touch was applied to merge similar definitions together where it was possible to do so with punctuation only (Abdication and Abdomen) and the definitions would feel redundant when placed side-by-side.

I uploaded a scan of the San Francisco Newletter and California Advertiser received from the Bancroft library. The Demon's Dictionary entries were then transcribed from the OCR version available at archive.org, then hand-corrected and verified.

I added the additional source to `content.opf` but did not modify the imprint.

A cropped scan of the Demon's Dictionary column is available on
WikiMedia:
https://commons.wikimedia.org/wiki/File:First_Demon%27s_Dictionary_Column_in_the_San_Francisco_Newletter_and_California_Advertiser_(Dec_11_1875).png

acabal commented:
Great, thanks! Two tweaks: We don't include Wiki links in the long description so you can remove that. Also, let's call it an "edition" and not a "version" as "version" has a kind of softwarey feel. Thanks!

mmastrac commented:
Awesome, thanks again. I squashed the recommended changes into the first commit (version -> edition and removed link). 

```diff
-                       &lt;p&gt;In 1911 the &lt;i&gt;Devil’s Dictionary&lt;/i&gt; as we know it was published in complete form as part of Bierce’s collected works (volume 7 of 12), including the remainder of the definitions from M to Z. It has been republished a number of times, including more recent efforts where older definitions from his columns that never made it into the original book were included. Due to the complex nature of copyright, some of those found definitions have unclear public domain status and were not included. This edition of the book includes, however, a set of definitions attributed to his one-and-only “Demon’s Dictionary” column, including Bierce’s classic definition of A: “the first letter in every properly constructed alphabet.”&lt;/p&gt;
+                       &lt;p&gt;In 1911 the &lt;i&gt;Devil’s Dictionary&lt;/i&gt; as we know it was published in complete form as part of Bierce’s collected works (volume 7 of 12), including the remainder of the definitions from M to Z. It has been republished a number of times, including more recent efforts where older definitions from his columns that never made it into the original book were included. Due to the complex nature of copyright, some of those found definitions have unclear public domain status and were not included. This version of the book includes, however, a set of definitions attributed to his one-and-only &lt;a href="https://commons.wikimedia.org/wiki/File:First_Demon%27s_Dictionary_Column_in_the_San_Francisco_Newletter_and_California_Advertiser_(Dec_11_1875).png"&gt;“Demon’s Dictionary” column&lt;/a&gt;, including Bierce’s classic definition of A: “the first letter in every properly constructed alphabet.”&lt;/p&gt;
```

acabal commented:
Thanks!
